src/strategy/strategy.ts

```diff
@@ -17,6 +18,15 @@ export interface Constraint {
   caseInsensitive?: boolean;
 }
 
+export interface Segment {
+  id: number,
+  name: string,
+  description?: string,
+  constraints: Constraint[],
+  createdBy: string,
+  createdAt: Date
```

Comment:
Should `createdAt` be a string or do we automatically parse dates somewhere?

Reply:
Awesome, it's a string now!